### PR TITLE
Remove un-needed function

### PR DESCRIPTION
**Pull request by ChrisAHolland**
##### SUMMARY

Comments explaining this function state:
```
# FIXME: Remove this once this PR is merged:
#   https://github.com/ansible/ansible/pull/19172
```
And it so appears that the PR has been merged some time ago, so let's reduce unnecessary code!

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

`modules/packaging/os/dnf.py`

### CODE REVIEW

**Review by mkrizek — CHANGES_REQUESTED** [review]
That method is still used:
https://github.com/ansible/ansible/blob/c61f141bbc97e0d17dffc7da5e560e8786613b6c/lib/ansible/modules/packaging/os/dnf.py#L795

Please see the pull request you linked on how the method was replaced in other packaging modules.

**Comment by ChrisAHolland** [comment]
Everything should be good now!

**Comment by mkrizek** [comment]
With the method removed there are now some unused imports which should be removed as well.

**Comment by mkrizek** [comment]
> @ChrisAHolland this PR contains the following merge commits:
> 
>     * [1bb78cb](https://github.com/ansible/ansible/commit/1bb78cba2d17e4080dfbe268975fd46726bc5d5c)
> 
> 
> Please [rebase your branch](http://docs.ansible.com/ansible/devel/dev_guide/developing_rebasing.html) to remove these commits.
> 
> [click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

This needs to be resolved as well.

**Comment by ChrisAHolland** [comment]
> With the method removed there are now some unused imports which should be removed as well.

I removed the 1 import for `fetch_url`. That was the only one I believe.

**Comment by mkrizek** [comment]
Also `tempfile` is unused now.

**Comment by ChrisAHolland** [comment]
> Also `tempfile` is unused now.

Now removed.

File: changelogs/fragments/68482-remove-function-update-calls.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- Removed lib/ansible/modules/packaging/os/dnf.py meant to be removed once PR 19172 was merged and updated function calls.
```
Comment (mkrizek) [inline changelogs/fragments/68482-remove-function-update-calls.yaml]: I am not sure this makes sense.

**Comment by mkrizek** [comment]
Code looks good to me now.

File: changelogs/fragments/68482-remove-function-update-calls.yaml (hunk 1 above)
Comment (ChrisAHolland) [reply changelogs/fragments/68482-remove-function-update-calls.yaml]: Updated it. Hopefully everything is good now :)

**Comment by ChrisAHolland** [comment]
@mkrizek @maxamillion Any updates? I'm excited to have my first PR merged :) Thanks!

**Comment by ChrisAHolland** [comment]
ready_for_review

**Comment by ChrisAHolland** [comment]
@mattclay @bcoca Could I get a review and merge on this PR? Sorry for being annoying :P

File: changelogs/fragments/68482-remove-function-update-calls.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+bugfixes:
+- Removed deprecated function from lib/ansible/modules/packaging/os/dnf.py meant to be removed once PR 19172 was merged.
```
Comment (mkrizek) [inline changelogs/fragments/68482-remove-function-update-calls.yaml]:
```suggestion
- dnf - remove custom ``fetch_rpm_from_url`` method in favor of more general ``ansible.module_utils.urls.fetch_file``.
```

File: changelogs/fragments/68482-remove-function-update-calls.yaml [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+- Updated function calls to most current function and removed unneeded imports.
```
Comment (mkrizek) [inline changelogs/fragments/68482-remove-function-update-calls.yaml]:
```suggestion
```

This level of detail is unnecessary.

**Comment by mkrizek** [comment]
The lines 309 and 310 are now unused as well:
```python
# 64k.  Number of bytes to read at a time when manually downloading pkgs via a url
BUFSIZE = 65536
```

**Comment by ChrisAHolland** [comment]
@mkrizek Made the changes! I appreciate your help!

**Review by mattclay — APPROVED** [review]

**Review by mkrizek — APPROVED** [review]

**Comment by ChrisAHolland** [comment]
Thanks for the help and merge!

**Pull request merged**